content/concept-amendments.md

```diff
@@ -130,11 +133,21 @@ The following is a comprehensive list of all known amendments and their status o
 
 | Amendment ID                                                     | Status  |
 |:-----------------------------------------------------------------|:--------|
-| 1562511F573A19AE9BD103B5D6B9E01B3B46805AEC5D3C4805C902B514399146 | In development |
+| 1562511F573A19AE9BD103B5D6B9E01B3B46805AEC5D3C4805C902B514399146 | Enabled |
 
-Adds conditional payments that are directly compatible with the [Interledger Protocol Crypto-Conditions specification](https://tools.ietf.org/html/draft-thomas-crypto-conditions-02).
+Although this amendment is enabled, it has no effect unless the [SusPay](#suspay) amendment is also enabled. Ripple plans to withdraw SusPay and implement the same functionality as part of the [Escrow](#escrow) amendment including crypto-condition support.
+
+
+## Escrow
+
+| Amendment ID                                                     | Status  |
+|:-----------------------------------------------------------------|:--------|
+| 07D43DCE529B15A10827E5E04943B496762F9A88E3268269D69C44BE49E21104 | In Development |
+
+Replaces the [SusPay](#suspay) amendment, incorporating functionality from the [CryptoConditions](#cryptoconditions) amendment.
+
+Provides "suspended payments" for XRP for escrow within the Ripple Consensus Ledger including support for [Interledger Protocol Crypto-Conditions](https://tools.ietf.org/html/draft-thomas-crypto-conditions-02). Creates the `SuspendedPayment` ledger node type and the new transaction types `SuspendedPaymentCreate`, `SuspendedPaymentFinish`, and `SuspendedPaymentCancel`.
```

Comment:
Should there be a comma before including?

Also, we will likely be renaming the code to use different names for the transactions and the ledger node: potentially EscrowCreate, EscrowFinish and EscrowCancel but the names are not finalized yet. Maybe we can punt on this last sentence until that time?